verification/src/http_server/routers.rs

```diff
@@ -0,0 +1,26 @@
+mod app;
```

Comment:
let's have one style everywhere: `module/mod.rs` or `module.rs`.

Reply:
Agree. Personally I prefer the second style as it seems to be more descriptive. But that requires to change all other modules. Changed `routers.rs` to `routers/mod.rs` for now, and suggest to discuss style changing at another issue